content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -110,6 +111,29 @@ either `Delete` or `Retain`. If no `reclaimPolicy` is specified when a
 Persistent Volumes that are created manually and managed via a storage class will have
 whatever reclaim policy they were assigned at creation.
 
+### Allow Volume Expansion
+
+{{< feature-state for_k8s_version="v1.15" state="beta" >}}
+
+Persistent Volumes can be configured to be expandable. This feature when set to `true`, 
+allows the users to resize the volume by editing the corresponding PVC object. 
+
+The following types of volumes support volume expansion, when the underlying
+Storage Class has the field `allowVolumeExpansion` set to true.
+
+* gcePersistentDisk
```

Comment:
@gnufied, can you verify which version all of these plugins were implemented in?

Reply:
ping @gnufied for confirmation :)

Reply:
here is support/version matrix of various volume types look like:

- EBS (1.11)
- Cinder (1.11)
- Glusterfs (1.11)
- GCE PD (1.11)
- Portworx (1.11)
- Azure File (1.11)
- RBD (1.11)
- Azure Disk(1.11)
- Flex Volume (1.13)
- CSI (1.14, 1.15 - Alpha, 1.16 beta)

Reply:
It might be worth updating this document and explicitly specify which version of k8s the volume plugin type supported expansion.